index.js

```diff
@@ -1,27 +1,67 @@
 var stylus  = require('stylus');
+var path    = require('path');
+var fs      = require('fs');
+
+var stylusUtils = require('stylus/lib/utils');
+
+var stylusLookupIndex = stylusUtils.lookupIndex;
+
+stylusUtils.lookupIndex = function(name, paths, filename){
+  var nodeModuleMatch = paths.find(includePath => includePath.endsWith(name));
+
+  var found = stylusLookupIndex(name, paths, filename);
+  
+  // check for node_modules/<name>
+  if (!found && nodeModuleMatch) {
+    found = stylusLookupIndex(nodeModuleMatch, paths, filename);
+  }
+  if (!found) {
+    console.warn('[derby-stylus] Could not find ' + name + ' in ' + paths.join(', '));
+  }
+  return found;
+};
+
+
+module.exports = function(app, options = {}) {
+  var { includes = [], plugins = [], ...opts } = options;
```

Comment:
It looks like `opts` is intended to be a base set of options to be passed directly to the Stylus compiler?

If that's the case, I'd prefer them to be passed as an `stylusOptions` property for clarity, instead of intermingled with the Derby plugin options.

Can also name the local var to emphasize that it's a base set of Stylus options, since it can be overridden later by individual `loadStylesSync` calls (also see comment further below)
```js
var baseStylusOptions = options.stylusOptions;
...
function stylusCompiler(file, filename, options) {
  var options = { _imports: [], ...baseStylusOptions, ...options };
  ...
```